src/sql/workbench/common/editor/query/queryEditorInput.ts

```diff
@@ -106,6 +106,14 @@ export class QueryEditorState extends Disposable {
 	public get isSqlCmdMode(): boolean {
 		return this._isSqlCmdMode;
 	}
+
+	public setState(oldState: QueryEditorState): void {
```

Comment:
Shouldn't this be `newState`?

Reply:
We're updating this with the state of the old editor, but since it is technically a new state, I'll change it to that.